core-primitives/stf-interface/src/lib.rs

```diff
@@ -33,6 +33,8 @@ pub mod parentchain_pallet;
 pub mod sudo_pallet;
 pub mod system_pallet;
 
+pub const SHARD_VAULT_KEY: &str = "ShardVaultPubKey";
```

Comment:
technical debt. we may actually solve it differently: https://github.com/integritee-network/pallets/issues/227